clients/drcachesim/tools/view.h

```diff
@@ -87,7 +87,7 @@ class view_t : public analysis_tool_t {
     inline void
```

Comment:
As I warned, this breaks tests with golden output like `tool.drcacheoff.view`: as you can see, several suites are red.  Normally you wouldn't request a review until all tests pass (or possibly you point at filed issues for any failues considered flakes).  Please fix.